src/AppBundle/Tests/DataFixtures/ORM/posts.yml

```diff
@@ -0,0 +1,10 @@
+AppBundle\Entity\Post:
+    post1:
+        title: 'Clean Code principles.'
+        author: 'Robert C. Martin'
+    post2:
+        title: 'TDD in practise.'
```

Comment:
I think it should be "practice" => http://www.world-english.org/practise_practice.htm